configure.ac

```diff
@@ -117,4 +114,32 @@ AS_IF([test "x$with_docs_only" = xyes],
     ]) # Docs only
 
 
+AC_ARG_ENABLE([debug],
+              AS_HELP_STRING([--enable-debug], [Enable extra debugging code (default is NO).]),
+              [], [enable_debug=no])
+
+if test $enable_debug = yes; then
+	AC_DEFINE([ENABLE_DEBUG], [1], [Enable debugging code])
+	CFLAGS="$CFLAGS -O0 -g3"
+	AC_DEFINE([UCS_MAX_LOG_LEVEL], [UCS_LOG_LEVEL_TRACE_POLL], [Highest log level])
```

Comment:
Why compile-time if we can do run-time, as part of params?

Reply:
@alex--m UCS logger requires UCS_MAX_LOG_LEVEL macro defined during build time. So, we have to define it to some value. We 100% followed UCX logic here. Enabling high max log level in debug build (TRACE) and lower level (INFO) for release build.